.vscode/settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
this file doesnt need to be committed. After you remove it you can add it to the gitignore